docs/docs/api/useAnimatedProps.md

```diff
@@ -11,6 +11,11 @@ The provided worklet is responsible for returning a object with view properties.
 
 Only "native" properties of "native views" can be set via `useAnimatedProps`.
 The most common usecase for this hook is when we want to animate properties of some third-party native component, since most of the properties for the core React Native components are a part of the styles anyways (at least the properties for which it makes sense to be animated).
+You can use following functions to animate properties that Reanimated don't support by default:
+
+1. `addWhitelistedNativeProps()` is used to animate properties that triggers layout recalculation, you can find them [here](https://github.com/software-mansion/react-native-reanimated/blob/master/src/ConfigHelper.js#L31).
+
+2. `addWhitelistedUIProps()` is used for any other properties, current allowed props are listed [here](https://github.com/software-mansion/react-native-reanimated/blob/master/src/ConfigHelper.js#L6).
```

Comment:
+ I'm not sure if 
> any other properties

is a good description here. I would rather go for something like "properties that are updated directly on the UI thread".

Reply:
Fixed in cc4fe25. Thanks for the suggestions.